lib/Config.php

```diff
@@ -151,10 +151,8 @@ public static function setSystemConfiguration(?array $configuration, string $off
     public static function getSystemConfiguration(string $offset = null): ?array
     {
         if (null === static::$systemConfig && $container = Pimcore::getContainer()) {
-            $config = $container->getParameter('pimcore.config');
```

Comment:
This breaks the admin session bag now 
![image](https://user-images.githubusercontent.com/5137917/233677443-d51fe5d9-8da1-46cb-9382-4ebaf0006312.png)